src/main/java/tira/Ui.java

```diff
@@ -0,0 +1,74 @@
+package tira;
+import tira.task.TaskList;
+import tira.task.Event;
+import tira.task.Task;
+import tira.task.ToDo;
+import tira.task.Deadline;
+
+import java.util.Scanner;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+/*
+Ui class
+Handles the user's commands, including reading the user commands
+Handles printing of all the errors.
+Used as an input for TaskList, Storage, and Tira Main.
+ */
+public class Ui {
```

Comment:
Perhaps it would be neater to use line breaks consistently between each segment of the code, for example between all method code blocks, rather than only some of them.